aten/src/ATen/native/cuda/SortStable.cu

```diff
@@ -288,4 +290,17 @@ void launch_stable_sort_kernel(
       });
 }
 
+void launch_stable_sort_kernel(
+    const TensorBase& self,
+    int64_t dim,
+    bool descending,
+    const TensorBase& values,
+    const TensorBase& indices) {
+
+  AT_DISPATCH_V2(indices.scalar_type(), "launch_stable_sort_kernel", AT_WRAP([&] {
+    scalar_t* indices_ptr = indices.mutable_data_ptr<scalar_t>();
+    do_launch_stable_sort_kernel(self, dim, descending, values, indices_ptr);
+  }), kByte, kUInt16, kUInt32, kLong);
```

Comment:
You can change the `kByte, kUInt16, kUInt32, kLong` into `kUInt16, kLong` or `kLong` for testing

Reply:
Plus, you should update `cub-RadixSortPairs-*.cu` for exact testing, removing unnecessary type.